multirun/def.bzl

```diff
@@ -26,7 +26,17 @@ def _multirun_impl(ctx):
     runfiles = runfiles.merge(runnerInfo.default_runfiles)
 
     commands = []
+    labeled_commands = []
```

Comment:
Some items in this list have `label` as type `string` and some as type `Label`. I think this might lead to confusion. I suggest to make them all strings and also change the name of the field to `tag` to match how it's called below.

Reply:
Right, shall we go with `tagged_commands` then to avoid any confusion with bazels use of `label`?

Reply:
Makes sense